mlflow/gateway/config.py

```diff
@@ -34,6 +34,7 @@ class Provider(str, Enum):
     OPENAI = "openai"
     ANTHROPIC = "anthropic"
     COHERE = "cohere"
+    AI21LABS = "ai21labs"
```

Comment:
can we add a minimum example in https://github.com/mlflow/mlflow/tree/master/examples/gateway?

Reply:
we can add an example in a follow-up PR.

Reply:
Sg, I will add docs and example in the next PR!